charts/auto-loans/origination-activity/figure-1a__number.html

```diff
@@ -0,0 +1,13 @@
+    
+<p class="block short-desc">
+    <strong>Source:</strong> <a href="https://github.com/cfpb/consumer-credit-trends/blob/master/data/num_data_AUT.csv">https://github.com/cfpb/consumer-credit-trends/blob/master/data/num_data_AUT.csv</a><br>
+    <strong>Date published:</strong> month year<br>
+    <strong>Note:</strong> Data from the last 6 months is not final and is lorem ipsum dolor sit.
```

Comment:
Since we haven't received any content for the Lorem Ipsum piece, can we remove it? So we don't accidentally ship lorem ipsum to production. :trollface:

So, `<strong>Note:</strong> Data from the last 6 months is not final.`